ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -150,6 +150,32 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-byo-route53
+- as: aws-ipi-disconnected-cco-manual-sts-w-ep-p2-f7
```

Comment:
How about rename it to `aws-ipi-disconnected-cco-manual-sts-endpoint-p2-f7`

Reply:
`aws-ipi-disconnected-cco-manual-sts-endpoint-p3-f28-destructive` exceeds the maximum length limitation.
How about `aws-ipi-disconnected-cco-manual-sts-ep-p3-f28-destructive`?

Reply:
ah, forget the `destructive` one. Yeah, `aws-ipi-disconnected-cco-manual-sts-ep-p3-f28-destructive` sounds good to me.

Reply:
Updated.